build-configs/aschaffenburg/theme/fonts.js

```diff
@@ -4,14 +4,17 @@ import type { FontsType } from '../../common/theme/fonts'
 import { commonFonts } from '../../common/theme/fonts'
 
 const fonts: FontsType = {
-  decorativeFontBold: 'VarelaRound-Regular',
-  decorativeFontRegular: 'VarelaRound-Regular',
-  decorativeFontFamily: '\'Varela Round\', \'Lateef\', sans-serif',
-  contentFontRegular: 'OpenSans-Regular',
-  contentFontBold: 'OpenSans-Bold',
-  contentFontFamily: '\'Open Sans\', \'Lateef\', sans-serif',
-  arabicWebviewFontFamilies: '\'OpenSans\', \'Lateef\', sans-serif',
-  webviewFontFamilies: '\'OpenSans\', sans-serif',
+  native: {
+    decorativeFontBold: 'VarelaRound-Regular',
```

Comment:
Does this work on iOS for you? App crashing for me here.

Reply:
not part of this task :P